internal/providers/terraform/azure/linux_virtual_machine.go

```diff
@@ -0,0 +1,74 @@
+package azure
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/infracost/infracost/internal/schema"
+
+	"github.com/shopspring/decimal"
+)
+
+// todo: add devtest consumption
```

Comment:
Should this be mentioned in Notes below?

Reply:
Sorry, I should have removed this todo. Will update.